tooling/tracer/src/debugger_glue.rs

```diff
@@ -42,8 +42,9 @@ fn convert_debugger_stack_frame(
     debugger_stack_frame: &noirc_artifacts::debug::StackFrame<FieldElement>,
 ) -> StackFrame {
     let function_name = String::from(debugger_stack_frame.function_name);
-    // TODO(stanm): add call params and values
-    StackFrame { function_name }
+    // TODO(stanm): add call params
+    let variables = debugger_stack_frame.variables.iter().map(Variable::from_tuple).collect();
+    StackFrame { function_name, variables }
```

Comment:
Does the function convert from StackFrame to StackFrame?

Reply:
It does, but between different `StackFrame`s. From `noirc_artifacts::debug::StackFrame` to `crate::stack_frame::StackFrame`.

Any suggestions?